src/components/molecules/SfGallery/SfGallery.html

```diff
@@ -0,0 +1,20 @@
+<div class="sf-gallery">
+  <div class="sf-gallery__nav">
+    <div v-for="(image, i) in images"
+      :key="'img-' + i"
+      class="sf-gallery__item"
+      :class="{ 'sf-gallery__item--selected': i === activeIndex }"
+      @click="setCurrentImage(i)">
+      <slot name="thumb" v-bind:image="image">
+        <img class="sf-gallery__thumb" :src="image.small.url" :alt="image.small.alt"/>
+      </slot>
+    </div>
+  </div>
+  <div class="sf-gallery__stage" v-if="currentImage">
```

Comment:
isn't "current" more descriptive

Reply:
actually, I'm already using `current` as prop (the default selected image index), so I would also have to change the prop name.
maybe `active` for the prop name and `current` for the current image object, you think is better?
I used `current` on the prop name because the pagination component also the same prop, so I was trying to maintain standards.

Reply:
Maybe a sync task since both current and active are refering to same think. Initial state of current is active right?

Reply:
On my implementation, `current` is a number (the image index + 1, so 1 for the first image), and `currentImage` is the object of the selected image, so they're not the same thing.
Even though, sync is already working for the `current` prop.

Reply:
I can avoid using the image object, and explicitly use `images[current - 1]`...

Reply:
Don't you think it's bad to repeat `images[current - 1]` many times?
```html
<slot name="current" v-bind:image="images[current - 1]">
    <img class="sf-gallery__big-image"
      :src="images[current - 1].big.url"
      :alt="images[current - 1].big.alt" />
  </slot>
```

In addition, if I use the property (`current`) directly, the component will work only with `v-sync`, using the computed `currentImage` as I've implemented, the component may work with or without sync, is optional, don't you think it's better?